pkg/marathon/client.go

```diff
@@ -64,5 +65,31 @@ func (c *Client) GroupsWithoutRootSlash() (map[string]bool, error) {
 	default:
 		return nil, errors.New("unable to get Marathon groups")
 	}
+}
+
+// Info returns the content of the Marathon endpoint 'v2/info'.
+func (c *Client) Info() (map[string]interface{}, error) {
+	dcosClient := pluginutil.HTTPClient(c.baseURL)
+	resp, err := dcosClient.Get("/v2/info")
+	if err != nil {
+		return nil, err
```

Comment:
Please add more context to this error

Reply:
The error reported by the HTTPClient should give enough information and this is how GET calls are handled in the rest of the codebase (e.g. https://github.com/dcos/dcos-core-cli/blob/2bc9f1b3f61fe50a4c863ae2f176c30815d473b8/pkg/diagnostics/v2/client.go#L53). Could you tell me what kind of information should I add?

Reply:
OK let's leave it for now. This is a prat of bigger discussion how to handle errors in CLI